### ISSUE TITLE
Sexual propensities between consenting adults

### ISSUE THREAD

**Issue by CoralineAda** [ContributorCovenant/contributor_covenant#392]
The point of [this PR](https://github.com/ContributorCovenant/contributor_covenant/pull/391) is to prevent already marginalized people from discrimination based on kink or sexual proclivities.

This is a large enough and controversial enough topic that I would like to open it up for public discussion. Be sure to be respectful and remember our own CoC in these discussions.

**Comment by aredridel** [comment]
This one is tricky, I think, because a simple clause to poke it makes a nice foothold to justify sexual behavior -- especially ambiguously so -- in spaces. In the absence of an enforcement document, I'm not sure what the best way to address it is.

**Comment by juliepagano** [comment]
It seems like most of the cases I can think of that would be a worry around this are covered by other parts of the CoC already. E.g. if someone was outed for a thing they keep out of a professional setting, that would be covered by the section about publishing others private information; discussions of sexual details inappropriate for a professional setting would be covered by the section about sexualized language/imagery; and the mention of sexual identity/orientation might cover a few things. 

Do you have some use cases you (or others) feel aren't covered by the current code of conduct that you're concerned about adding coverage for?

**Comment by wirehead** [comment]
This has been on my mind, because I've applied the CC CoC to one of my projects and it's got good image support, which means that it's almost inevitable that it it's going to be used for a person's kink, if people start to use it.

Let me see if I can try to enumerate some cases that came to my mind:

If someone's using the software to power their kink, that's covered by the overall software license and saying "...except porn" makes it fail most free software / open source license standards.  And that doesn't touch the CoC because it's all about contributions.

If someone were to report a bug encountered while in the service of a kink, they'd have to carefully structure their bug or other sort of contribution to make sure that it was SFW.  That's already covered, I think, by the unacceptable behavior section.

If a person has a kink on their own twitter/facebook/myspace/et al account and keeps a valid separation between that account/accounts and their public profile, that's covered by scope, I think.  Outing someone and violating that separation is also covered.

You want to avoid the case where a contributor who has a kink is quietly boundary-pushing.  If both sides obey the CoC, the "Project maintainers are responsible for clarifying..." cause makes me think that when it's a surprise to the contributor, that means that the maintainers haven't been clarifying.  But I think that's covered by the "other conduct..." section to be a boundry-pushing kinkster.

And then you have, as @CoralineAda says, the desire to "protect already vulnerable people from discrimination based on kink or sexual proclivities", where a hypothetical member of the vulnerable population has a kink and another member seeks to remove said vulnerable member, solely on the basis of their kink.  This entire discussion, I've been trying to push the case that triggered this out of my head and think what would happen to the people I know with a kink who are women or of color or disabled.

Maybe there's the larger issue of having one's private life outside of the project used against them, of which kink is a subset?  As in, what would happen if one were to change "or sexual proclivities between consenting adults" in the 1.4.1 draft to "or private life"?  Maybe there's a better way to say that.  Because I'm thinking of other things that can be weaponized to exclude people.  Instead of just a kink, what about sailor moon fanfic?

**Comment by Starmann** [comment]
@wirehead I'd suggest being careful about the wording in the phrase "Personal life". It could be used to defend White Supremacists, for Instance.

**Comment by icambridge** [comment]
> @wirehead I'd suggest being careful about the wording in the phrase "Personal life". It could be used to defend White Supremacists, for Instance.

Just curious, is the point of this document to be able to exclude people you specifically don't like even if they do it privately without others knowing?

>  or sexual proclivities between consenting adults.

Also, who is to decide who and who is not consenting, because I saw a tweet from @CoralineAda where she decided that people couldn't consent to something even though the law says they can. So is this to allow for the ability to decide who is consenting and who is not and exclude those kinks that you dislike?

Also, there seems to be a lot of talk about making sure vulnerable people are protecting, so does this document/clause only protect those defined as vulnerable and others are on their own or can be removed at will? Or is it to create a space that everyone can be welcome to as long as they publically adhere to a specific set or rules?

**Comment by SuperintendentColeman** [comment]
Switch to Federal-REACH LOGO PLEASE.

**Comment by victimsofgaybullying** [comment]
#538 

Shows this document is already being used to say you must respect those who prefer to have sex between men and little boy bums on forums away from open-source project infrastructure or you will be proactively banned on lets say freedesktop.org for example.

So yeah you better say "consenting adults" instead of "sexual preferences".

My friend was found guilty of the crime of linking to http://tinyurl.com/abcplugspedos

Put "abcplugspedos" into www.caselaw.nsw.gov.au or austlii.edu.au to see what I mean or "mckee burns" to look up my dad's case being found guilty of the crime of hating sex between state media praised gay marriage Activists Mark Newton and Peter Truong and Russian infant Drake Newton. That's right the gay mafia came after my dad in Australia because they couldn't get me abroad as a UNHCR confirmed political refugee wanted for Australia's rarest crime 60C NSW Crimes Act, a law that the Greens Gay Rights Spokesperson Lee Rhiannon said in Hansard only corrupt police would use before only special gay police (facebook.com/policeglbti also twitter.com/glludc)  did use it - against me.

https://www.youtube.com/watch?v=UHrR9MY6GdM

http://www.truthnews.com.au/web/radio/story/cultural_marxism_101

https://voat.co/v/pizzagate/2018442

This shows you that gay anal sex normalization with infants through the act of criminalizing dissent against those who rape children is already happening in the courts, not just through the open-source movement.

A Month after this was posted #FreeStacy Happened:
http://theothermccain.com/2016/05/14/australian-gay-marriage-crusader-was-fugitive-wanted-on-kiddie-porn-charge/ ( I was the researcher )

The government if you see the issue #538 formally ruled child porn is OK if you are gay, by ruling it's not a character concern or blemish on your character if you download child porn and say you are gay and hug the mayor of brisbane before you are deported http://bernardgaynor.com.au/happy

I'm also be exposing more paedophilia normalization and a big human trafficking ring takedown later this month ran by jailed human trafficker praised by CNN Tammy Davis Charles:

https://153news.net/watch_video.php?v=M8K996WOHNXD

**Comment by victimsofgaybullying** [comment]
What was wrong with the deleted comment I screenshot? Did it disrespect pedophiles and that was wrong mmmkay @CoralineAda 

You know that you are in Oregon. You must respect Democrat King Pin Terry Bean charged with child rape rules the Australian court too! Here's the details.

https://www.quora.com/Who-would-you-trust-alone-with-your-kids-Obama-or-Trump/answer/Luke-McKee-1

This the case law where you can see a picture in case law of Terry Bean and Obama HRC co-founder charged with sodomizing a minor and details on where you can read the minor's affidavit before he was disappeared along with his family for the sake of gay pride.

I've also got a lot of research about the OTO.org child sex cult's activities in Oregon. If you care about child safety not having them raped we would really work together.

https://www.caselaw.nsw.gov.au/decision/569ec29ee4b05f2c4f04aa7d

But you'll delete this screenshotted comment if you like child rape, because it' wrong to shame those who are charged with having sex with children right? Especially if they are top tier "queer hero" gay marriage activists.

In other words you are saying sex with children is OK as long as you say the magic word "I'm gay" first if you delete this comment.

**Comment by Chealer** [comment]
@CoralineAda : Please clarify which issue this tracks
